sirepo/template/srw.py

```diff
@@ -1628,7 +1603,7 @@ def _generate_parameters_file(data, plot_reports=False, run_dir=None):
     dm = data.models
     # do this before validation or arrays get turned into strings
     if is_for_rsopt:
-        rsopt_ctx = _rsopt_jinja_context(dm.exportRsOpt)
+        rsopt_ctx = _rsopt_jinja_context(data.report, dm.exportRsOpt)
```

Comment:
see above. _SIM_DATA.lib_file_basenames(data) needed here too? If not, pass in a conditional to not add it to the context.